packages/messaging/android/src/main/java/io/invertase/firebase/messaging/ReactNativeFirebaseMessagingStore.java

```diff
@@ -2,10 +2,12 @@
 
 import com.google.firebase.messaging.RemoteMessage;
 
+import com.facebook.react.bridge.WritableMap;
+
 public interface ReactNativeFirebaseMessagingStore {
   void storeFirebaseMessage(RemoteMessage remoteMessage);
 
-  RemoteMessage getFirebaseMessage(String remoteMessageId);
+  WritableMap getFirebaseMessage(String remoteMessageId);
```

Comment:
This is a breaking change if done like this - is it possible to maintain the original symbol, mark it as deprecated, and make new symbol `getFirebaseMessageMap`?

Reply:
fixed